peoplepage.html

```diff
@@ -77,138 +103,141 @@
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.3.0/js/bootstrap.bundle.min.js"></script>
+    <script>
+        const apiKey = 'aadadbbda0dacb0d2cb893bbad0a7d9d';
+        const apiUrl = 'https://api.themoviedb.org/3/person/popular?api_key=' + apiKey;
+        const artistsContainer = document.getElementById('artistsContainer');
+        const searchForm = document.getElementById('searchForm');
+        const searchBox = document.getElementById('searchBox');
+        const prevButton = document.getElementById('prevButton');
+        const nextButton = document.getElementById('nextButton');
+        let currentPage = 1;
+        let currentQuery = '';
+        let allArtists = [];
+
+        function filterArtists(query) {
+            if (query === '') {
+                return allArtists;
+            } else {
+                return allArtists.filter(artist => artist.name.toLowerCase().includes(query.toLowerCase()));
+            }
         }
-    }
-
-    function displayArtists(artists) {
-        artistsContainer.innerHTML = '';
-
-        if (artists.length === 0) {
-            const artistNotFoundMessage = document.createElement('p');
-            artistNotFoundMessage.textContent = 'Artist not found.';
-            artistsContainer.appendChild(artistNotFoundMessage);
-        } else {
-            let row = document.createElement('div');
-            row.classList.add('row');
-
-            artists.forEach((artist, index) => {
-                const artistCard = document.createElement('div');
-                artistCard.classList.add('col-md-2', 'artist-card');
-
-                const profileImage = document.createElement('img');
-                profileImage.src = `https://image.tmdb.org/t/p/w200${artist.profile_path}`;
-                profileImage.alt = artist.name;
-
-                const artistName = document.createElement('p');
-                artistName.textContent = artist.name;
-                artistName.classList.add('artist-name');
-
-                artistCard.appendChild(profileImage);
-                artistCard.appendChild(artistName);
-
-                artistCard.addEventListener('click', () => {
-                    const artistId = artist.id;
-                    const artistInfoUrl = `https://api.themoviedb.org/3/person/${artistId}?api_key=${apiKey}`;
-                    window.location.href = `artist.html?id=${artistId}&url=${artistInfoUrl}`;
-                });
 
-                row.appendChild(artistCard);
+        function displayArtists(artists) {
+            artistsContainer.innerHTML = '';
+
+            if (artists.length === 0) {
+                const artistNotFoundMessage = document.createElement('p');
+                artistNotFoundMessage.textContent = 'Artist not found.';
+                artistsContainer.appendChild(artistNotFoundMessage);
+            } else {
+                let row = document.createElement('div');
+                row.classList.add('row');
```

Comment:
suggest to add  a justify-content-center class here to center the artists on the page.

Reply:
done